### PR TITLE
Analyser: don't optimise the order of variables and equations

### LINKED ISSUE #1089: Analyser: don't optimise the order of variables and equations
In order to make our code generation "prettier", the analyser sorts its internal variables and equations. However, with some "proper" models this means wasting more than 1 second for no good reason. For instance, the generated code will list variables according to their type and in alphabetical order, but this is really pure aeshetic and therefore not needed.

### PR DESCRIPTION
**Pull request by agarny**
Fixes #1089.

### CODE REVIEW

**Comment by hsorby** [comment]
Was this [commit](https://github.com/cellml/libcellml/pull/1090/commits/ed7cbda364a60218d3f82c469b3ef9333ac377c9) added to the correct PR?

**Comment by agarny** [comment]
> Was this [commit](https://github.com/cellml/libcellml/pull/1090/commits/ed7cbda364a60218d3f82c469b3ef9333ac377c9) added to the correct PR?

Oops, you are completely right! Have just cherry picked it for PR #1080 (see commit 356e1707).

File: tests/coverage/coverage.cpp [hunk 1]
```diff
@@ -423,7 +423,8 @@ TEST(Coverage, generator)
     }
 
     for (size_t i = 0; i < analyserModel->variableCount(); ++i) {
-        EXPECT_EQ(i < 7, analyserModel->variable(i)->initialisingVariable() != nullptr);
+        EXPECT_EQ((i == 1) || (i == 2) || (i == 6) || (i == 18) || (i == 177) || (i == 178) || (i == 180),
+                  analyserModel->variable(i)->initialisingVariable() != nullptr);
```
Comment (hsorby) [inline tests/coverage/coverage.cpp]: Weird testing condition, I would prefer:
```
if ((i == 1) || (i == 2) || (i == 6) || (i == 18) || (i == 177) || (i == 178) || (i == 180)) {
    EXPECT_TRUE(analyserModel->variable(i)->initialisingVariable() != nullptr);
}
```

File: tests/resources/coverage/generator/model.implementation.out
Comment (hsorby) [inline tests/resources/coverage/generator/model.implementation.out:1]: I don't understand why this file has the suffix `.out`.

File: tests/resources/coverage/generator/model.out
Comment (hsorby) [inline tests/resources/coverage/generator/model.out:1]: I don't understand why this file has the suffix `.out`.

**Review by hsorby — CHANGES_REQUESTED** [review]

File: tests/resources/coverage/generator/model.implementation.out
Comment (agarny) [reply tests/resources/coverage/generator/model.implementation.out:1]: Just an `out`put file.

File: tests/resources/coverage/generator/model.out
Comment (agarny) [reply tests/resources/coverage/generator/model.out:1]: Just an `out`put file.

File: tests/coverage/coverage.cpp (hunk 1 above)
Comment (agarny) [reply tests/coverage/coverage.cpp]: ce85080f

**Review by nickerso — DISMISSED** [review]

**Review by nickerso — APPROVED** [review]

**Review by hsorby — APPROVED** [review]

**Pull request merged**